.github/workflows/build-push-docker.yml

```diff
@@ -39,23 +42,32 @@ jobs:
       - name: Login to GAR
         run: |
           gcloud auth configure-docker us-docker.pkg.dev -q
+      
+      - name: Login to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v3
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
 
-      # For now, we're only pushing to the internal registry
       - name: Build and push Docker image
         id: docker_build
         uses: docker/build-push-action@v6
         with:
           context: .
           push: ${{ github.event_name == 'push' && (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/')) }}
+          platforms: ${{ matrix.platform }}
           tags: |
             us-docker.pkg.dev/matterlabs-infra/matterlabs-docker/era-test-node:${{ steps.docker_tag.outputs.tag }}
             us-docker.pkg.dev/matterlabs-infra/matterlabs-docker/era-test-node:latest
```

Comment:
@popzxc how can we update this to be `anvil-zksync` instead of `era-test-node`?

Reply:
I would just try changing the name first and see if that will work. The way I am parsing this string is `matterlabs-infra` is the name of project, `matterlabs-docker` is the name of the container registry and `era-test-node` is the image in the registry. AFAIR you should be able to add new images to an existing registry by default unless we have disabled that on purpose.

Reply:
You can temporarily enable `on.push.branches.<your-branch-name>` to make this trigger before we merge the PR